### ISSUE TITLE
Can not get the performance in paper on nus with singe GPU 

### ISSUE THREAD

**Issue by codeman-24** [OpenPerceptionX/Openpilot-Deepdive#15]
I select the single gpu training as refered in the project.
# Training on a bare-metal machine with a single GPU
PORT=23333 SLURM_PROCID=0 SLURM_NTASKS=1 python main.py
but AP is low with AP@0.5(0-10) equas about 0.12.
How can i adjust it to reimplement your results on nus ?

my hyperparameters:
![image](https://user-images.githubusercontent.com/25739094/205044819-162a07ee-0a2b-4f18-a5dc-60850ae91484.png)
Thank you~

**Comment by ElectronicElephant** [comment]
Hi,

The low AP is expected as the batch size on single GPU is quite small. I think you can start with lowering the learning rate to 1e-5. I am not quite sure if it will work on single GPU. Good luck.

**Comment by codeman-24** [comment]
Thank you for ! I also anther question, the seq length used in nucenes is 18, How this parameter is set ？

**Comment by ElectronicElephant** [comment]
Hi,

The sequence length parameter is for training the GRU module, which means it will accumulate the gradients and make only one backward step every 18 batches.

It's not related with `Length Per Sequence`. LPS is only a property of the dataset.
